### PR TITLE
Automated deployment blog

### PR DESCRIPTION
**Pull request by shhetri**

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -39,3 +39,11 @@ manoj_byanjankar:
...
   avatar: default-thumb.png
   github: mojbenzcar
+
+sumit_chhetri:
+  name: Sumit Chhetri
+  bio: "Software Engineer"
+  avatar: sumit-avatar.png
+  github: shhetri
+  twitter: shhetri
+
```
Comment (geshan) [inline _data/authors.yml]: end of line

File: _posts/2015-09-05-automated-deployment.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+title: "Still stuck with FTP or Git pull to deploy? It's high time to embrace Automated Deployment with Capistrano"
+author: sumit_chhetri
+modified: 2015-09-05
+tags: [ deployment, devops, capistrano]
+---
+
+Are you still using FTP to deploy your code? Are you still manually performing the same steps of deploying a feature, again and again? How many hours have you spent on ssh-ing into the server, pulling the repo, migrating the database, reloading the web server and so on, for each deployment?? Ever wondered if there is a process as simple as a single click to perform all these steps for you?
+
+Automated Deployment does exactly these things for you. It takes the burden of remembering all the steps required in each deployment process and execute it smoothly.
+
```
Comment (geshan) [inline _posts/2015-09-05-automated-deployment.md:12]: picture here, one screenshot of deployment done is fine. with non verbose Capistrano output if possible.

File: _posts/2015-09-05-automated-deployment.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+<!--more-->
+
+You don't need the rocket sicence to conclude that machine is reliable, efficient and faster than humans. In context to the software development, be it writing your code, testing your code and deploying your code, it is no different story. You need to leverage the tools and technologies available, if you want to be a better software engineer.
```
Comment (geshan) [inline _posts/2015-09-05-automated-deployment.md]: `You don't need to know rocket science to conclude that machine is reliable`

Comment (geshan) [inline _posts/2015-09-05-automated-deployment.md]: It is not a different story. 

software engineer. If the tools are free that is even better.

File: _posts/2015-09-05-automated-deployment.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+In regard to software deployment, just try to count the number of times you deploy your software to staging, testing or production server, during its life cycle. I am pretty sure you don't have an exact number, or if you have, it is no less than hundreds. In each deployment you execute, it involves numerous steps; be it running some package installer scripts/commands, migrating your databases, notifying the clients about the deployment, and so on, which you will repeat again and again. You have no guarantee either, that the deployment will be successful each and everytime because YOU, a human, are deploying. Since you are deploying, it is more likely to be error prone than it would have been, had it been automated.
```
Comment (geshan) [inline _posts/2015-09-05-automated-deployment.md:19]: In regards to software...

the project's life cycle

a human, is deploying.

more error prone than automated.

File: _posts/2015-09-05-automated-deployment.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+I am going to deliver a presentation in a [PHP Dev Meetup #14](https://developers-nepal.github.io/php/meetups/05-Sep-2015/) on the very topic and below are the slides to that presentation. You can have a look at it and decide for yourself, whether you wanna keep getting stuck on the old fashioned and unreliable deployment process or you wanna go with the flow of the morden way of deploying.
```
Comment (geshan) [inline _posts/2015-09-05-automated-deployment.md]: in PHP...

on this topic 

slides of that presentation.

you want to stick to the old ...

or you want to follow modern deployment paradigm.

**Comment by geshan** [comment]
some changes and a rebase and you are ready to go.

**Comment by geshan** [comment]
@shhetri

File: _data/authors.yml (hunk 1 above)
Comment (shhetri) [inline _data/authors.yml]: there is a blank line dai

Comment (geshan) [inline _data/authors.yml]: ok

Warm Regards,
Geshan Manandhar

Blog http://geshan.com.np | Twitter http://twitter.com/geshan

**Comment by geshan** [comment]
Merging

**Pull request merged**